### PR TITLE
Manufacturing Capability Hierarchical Element Fix 3.1.0

### LINKED ISSUE #623: [Model Issue]: Manufacturing Capability Model HierarchyElement misconception
## URL to Model with Issue
https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.manufacturing_capability/3.0.0

## Issue Description 
The Manufacturing Capability Model has some issues regarding the representation of hierarchies.

1. The MachineToolEntity should not extend HierarchyElementAbstractEntity, instead MaterialFamilyEntity should.
2. Each Entity extending HierarchyElementAbstractEntity should be also extending ElementAbstractEntity. Thus, HierarchyElementAbstractEntity should extend ElementAbstractEntity.

The proposed changes can be found in https://github.com/FlorianPatzerC0d3/manufacturing-capability/tree/3.0.0/io.catenax.manufacturing_capability/3.0.0

Should this be adressed in a new version of the model or should the changes be merged into version 3.0.0, since the version 3.0.0 of the model is not in use yet with the exeption of a reference by the bill of process model (which will not be tangled by the changes).

### PR DESCRIPTION
**Pull request by FlorianPatzerC0d3**
## Description
As discussed with @agg3fe, we needed to fix the representation of hierarchical elements in the Manufacturing Capability Aspect Model. This fix is a minor update and pulls the version from 3.0.0 to 3.1.0.

Closes #623 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.5.1)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] If a previous model exists, model deprecation has been checked for previous model
- [x] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

File: io.catenax.manufacturing_capability/3.0.0/ManufacturingCapability.ttl
Comment (agg3fe) [inline io.catenax.manufacturing_capability/3.0.0/ManufacturingCapability.ttl:17]: Why are you making changes in 3.0.0 version when you are releasing 3.1.0 version?

File: io.catenax.manufacturing_capability/3.0.0/gen/ManufacturingCapability-schema.json
Comment (agg3fe) [inline io.catenax.manufacturing_capability/3.0.0/gen/ManufacturingCapability-schema.json:2]: you need to remove this file also. only folder 3.1.0 is required.

File: io.catenax.manufacturing_capability/3.0.0/ManufacturingCapability.ttl
Comment (FlorianPatzerC0d3) [reply io.catenax.manufacturing_capability/3.0.0/ManufacturingCapability.ttl]: With commit d11fae4ac9db536175e4a163f859d3e7f276162d there should not be any more changes in 3.0.0.

File: io.catenax.manufacturing_capability/3.0.0/gen/ManufacturingCapability-schema.json
Comment (FlorianPatzerC0d3) [reply io.catenax.manufacturing_capability/3.0.0/gen/ManufacturingCapability-schema.json]: Folder 3.0.0 should not contain any changes anymore. Does this make the comment obsolete?

**Comment by agg3fe** [comment]
@FlorianPatzerC0d3 Will keep this PR on hold for now, as cyclic dependency issue needs to be resolved first.

**Comment by agg3fe** [comment]
> Cyclic dependency: 3.1.0 of manufacturing capability model now references 1.1.0 from bill of process model, which will reference 3.1.0. Thus, with commit [d098b00](https://github.com/eclipse-tractusx/sldt-semantic-models/commit/d098b00a462136fc30b3cb0fe1d6fa3324ea7d81) the cyclic dependency is resolved as soon as bill of process model 1.1.0 is merged.

@FlorianPatzerC0d3 Thanks for the change You can raise the issue and PR for bill of process 1.1.0 also. I will do the MS2 for both PRs then.

**Review by agg3fe — APPROVED** [review]
LGTM

**Comment by agg3fe** [comment]
This PR will be merged after the merge of bill of process 1.1.0.

File: io.catenax.manufacturing_capability/RELEASE_NOTES.md
Comment (agg3fe) [inline io.catenax.manufacturing_capability/RELEASE_NOTES.md:5]: date 4th March

**Pull request merged**
